feat: add sync waves filter, auto generate tutor jobs - palm (#80)
eduNEXT/drydock: README.rst

We are defined by defult the following order:
- `All kubernetes resources` (except the ones that are defined in the next waves)
- `Initialization Jobs`
- `Upgrade Jobs`: When **DRYDOCK_MIGRATE_FROM** is set, over the Sync Wave 50
- `CMS and LMS Deployments`: When **DRYDOCK_POD_LIFECYCLE** is active, over the Sync Wave 100
- `Debug Resources`: When **DRYDOCK_DEBUG** active, over the Sync Wave 100
- `Horizontal Pod Autoscalers`: When active, over the Sync Wave 150

Workaround to upgrade from Maple to Palm
----------------------------------------

The upgrade from Maple to Palm fails because an issue with a squashed migrations in `edx-enterprise <https://github.com/openedx/edx-enterprise/blob/3.61.11/integrated_channels/blackboard/migrations/0001_initial_squashed_0014_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id.py>`_. To go around this issue, we need to apply migrations in a older version of edx-enterprise (3.60.4).

1. Run the sync to Palm without init jobs or upgrade jobs enabled.

> 2. Once with the LMS Deployment in the Palm version, go to one pod and run the following commands:

Reviewer: ```suggestion
2. Once the LMS Deployment is running in the Palm version, go inside a pod and run the following:
```
